### PR TITLE
SALTO-5660: Salesforce Quick Fetch logs

### PR DESCRIPTION
**Pull request by tamtamirr**
Add logs for Salesforce Quick Fetch

---

As part of this PR, also added a log in core that indicates we are running a quick-fetch for a specific account.

---
_Release Notes_: 
_None_

---
_User Notifications_: 
_None_

### CODE REVIEW

File: packages/salesforce-adapter/src/fetch_profile/metadata_query.ts [hunk 1]
```diff
@@ -245,11 +247,30 @@ export const buildMetadataQueryForFetchWithChangesDetection = async (
...
       instance.name,
     ])
-    return isValidDateString(dateFromSingleton) &&
-      isValidDateString(instance.changedAt)
-      ? new Date(dateFromSingleton).getTime() <
-          new Date(instance.changedAt).getTime()
-      : true
+    if (
+      !isValidDateString(dateFromSingleton) ||
+      !isValidDateString(instance.changedAt)
```
Comment (amitgoldarb) [inline packages/salesforce-adapter/src/fetch_profile/metadata_query.ts:252]: I find these if's a bit hard on the eyes, any chance we can keep this a positive, even if it's a bit longer in the end?

File: packages/salesforce-adapter/src/fetch_profile/metadata_query.ts [hunk 2]
```diff
@@ -245,11 +247,30 @@ export const buildMetadataQueryForFetchWithChangesDetection = async (
...
+    ) {
+      log.trace(
```
Comment (amitgoldarb) [inline packages/salesforce-adapter/src/fetch_profile/metadata_query.ts:254]: Can we possibly somehow aggregate these logs?

**Review by amitgoldarb — COMMENTED** [review]
Looks great!
Added a few comments

File: packages/salesforce-adapter/src/filters/changed_at_singleton.ts [hunk 3]
```diff
@@ -112,6 +116,10 @@ const filterCreator: LocalFilterCreator = ({ config }) => ({
           mostRecentChangedAt,
         )
       })
+    log.trace(
+      'ChangedAtSingleton values: %s',
+      safeJsonStringify(changedAtInstance.value),
```
[the remark this answers is not in the document]
Comment (tamtamirr) [reply packages/salesforce-adapter/src/filters/changed_at_singleton.ts:121]: I'm removing this log as I'm not sure it's required. There's other ways to access the `ChangedAtSingleton` instance if needed 🙏

**Pull request merged**